util/cookies.ts

```diff
@@ -0,0 +1,7 @@
+export const secureCookieOptions = {
+  httpOnly: true,
+  path: '/',
+  secure: process.env.NODE_ENV === 'production',
+  maxAge: 60 * 60 * 24, // This is 24 hours
+  sameSite: 'lax', // For cross site scripting
+} as const;
```

Comment:
This can be ordered like [the cookie options in the Next.js docs](https://nextjs.org/docs/canary/app/api-reference/functions/cookies#options) and documented better

You can use the version over here:

- https://github.com/upleveled/next-js-example-fall-2024-atvie/pull/10

Reply:
5952f2e1d31b05b729c4dce5cf09ce021e1b9bcd